### PR TITLE
Backdate VA parameter values to 2021

### LINKED ISSUE #3244: Backdate VA parameter values to 2021
For example personal and aged/blind exemption yaml files

### PR DESCRIPTION
**Pull request by Xinyi-Shao**
Fixes #3244

### Summary
📝📊🐛

This pull request updates and documents various parameters for Virginia income tax for 2021. It fixes the values of exemptions, deductions, subtractions, and rates based on the official sources. It also adds a changelog entry and a minor version bump.

> _Sing, O Muse, of the valiant code reviewers_
> _Who scrutinized the parameters of Virginia_
> _And updated them with care and skill, as heroes_
> _Who face the wrath of Python or Medusa_

### Walkthrough
*  Bump minor version and add changelog entry for fixed Virginia 2021 parameter values ([link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
*  Update standard deduction values for Virginia 2021 and add reference to source document (`standard.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-a512765c37f7dd684b66ca19e5e108d928aba632b0ef9c9800fb4bab554fd999L13-R30))
*  Update income tax brackets and rates for Virginia 2021 and add reference to source document (`rates.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-d0a3ca8ce00d9d5788efdb9d461cbe3a02f1718d5c7d7d1f989bfdfa0d276a35L12-R31))
*  Update income threshold for age deduction for Virginia 2021 and add reference to source document and code section (`threshold.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-f4c78122a65c2a02eb092322bc2fd58f7808eefd87f967228b41d19378344b94L14-R26))
*  Add birth year limit for full amount of age deduction for Virginia 2021 and add reference to source document (`birth_year_limit_for_full_amount.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-a364afe81c480f432c94a19b370613e939d416e1ead10155ed5a328590160893R4), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-a364afe81c480f432c94a19b370613e939d416e1ead10155ed5a328590160893R13-R14))
  - aged and blind exemption (`aged_blind.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-e645498315aef74d1e3a038950240122a706f0339ad37b52d678f2d68d102dabL3-R3), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-e645498315aef74d1e3a038950240122a706f0339ad37b52d678f2d68d102dabR14-R15))
  - personal exemption (`personal.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-e937e3606b68d51cb72d02dd62f9b1fdf17f08eaf95581c177f53e9e3235ed58L3-R3), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-e937e3606b68d51cb72d02dd62f9b1fdf17f08eaf95581c177f53e9e3235ed58R14-R15))
  - age minimum for age deduction (`age_minimum.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-36fa28aa5045820a6468d3619bce2f976d74c530ae466bef8f41fd2fb8d69e9eL3-R3), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-36fa28aa5045820a6468d3619bce2f976d74c530ae466bef8f41fd2fb8d69e9eR12-R13))
  - amount of age deduction (`amount.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-6d3776597fe4177b59d7178e8345eb1600abac08266bb0f1bee141431e646069L3-R3), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-6d3776597fe4177b59d7178e8345eb1600abac08266bb0f1bee141431e646069R13-R14))
  - amount of disability income subtraction (`disability_income/amount.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-31faf5a34a9cef4c3ee8fb7e80c4d21bf8fe80ce7648d93cacdf2cc66770a5bdL4-R4), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-31faf5a34a9cef4c3ee8fb7e80c4d21bf8fe80ce7648d93cacdf2cc66770a5bdR15-R16))
  - amount of federal and state employees subtraction (`federal_state_employees/amount.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-18f070d8451be63a81aa1eefe1feb664833ecb618b84c2403ec758f6e7e7f799L4-R4), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-18f070d8451be63a81aa1eefe1feb664833ecb618b84c2403ec758f6e7e7f799R15-R16))
  - threshold of military basic pay subtraction (`military_basic_pay/threshold.yaml`, [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-41239a17efe9ca8f0fe8b6a45a37fcc1846c98d754af44c0ae03ce385a1ee658L3-R3), [link](https://github.com/PolicyEngine/policyengine-us/pull/3268/files?diff=unified&w=0#diff-41239a17efe9ca8f0fe8b6a45a37fcc1846c98d754af44c0ae03ce385a1ee658R14-R15))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/va/tax/income/deductions/standard.yaml [hunk 1]
```diff
@@ -10,14 +10,21 @@ metadata:
     href: https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/
   - title: Virginia 2022 Form 760 Resident Individual Income Tax Booklet
     href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=18
+  - title: Virginia 2021 Form 760 Resident Individual Income Tax Booklet
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=18
 
 JOINT:
   2022-01-01: 16_000
+  2021-01-01: 9_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/va/tax/income/deductions/standard.yaml:18]: we put the 2021 variable first 
2021-01-01: 
2022-01-01: 

all parameter file

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    fixed:
+    - Updated Virginia 2021 parameter.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Updated Virginia 2021 parameter values.
```

File: policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 description: Virginia allows a full age deduction for taxpayers born before this year.
 values:
+  2021-01-01: 1957
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml:3]: just change to 2021-01-01: 1939

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml:3]: but please double check the structure to make sure that we calculate it correctly

File: policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/threshold.yaml [hunk 4]
```diff
@@ -11,14 +11,16 @@ metadata:
     href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=16
   - title: Code of Virginia § 58.1-322.03.(5)(a)(b)
     href: https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/
+  - title: Virginia 2021 Form 760 Resident Individual Income Tax Booklet
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=18
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/threshold.yaml:15]: double check the pages for the 2021 source

File: policyengine_us/parameters/gov/states/va/tax/income/deductions/standard.yaml [hunk 5]
```diff
@@ -10,14 +10,21 @@ metadata:
     href: https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/
   - title: Virginia 2022 Form 760 Resident Individual Income Tax Booklet
     href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=18
+  - title: Virginia 2021 Form 760 Resident Individual Income Tax Booklet
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=18
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/va/tax/income/deductions/standard.yaml:14]:
```suggestion
    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=19
```

File: policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml (hunk 3 above)
Comment (Xinyi-Shao) [reply policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml]: I have double checked the structure and the value should be 1939

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/va/tax/income/subtractions/age_deduction/birth_year_limit_for_full_amount.yaml]: can you change it?

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**